qiskit/quantum_info/synthesis/one_qubit_decompose.py

```diff
@@ -85,6 +88,10 @@ class OneQubitEulerDecomposer:
           - :math:`Z(\phi) Y(\theta) Z(\lambda)`
           - :math:`e^{i\gamma} U_1(\phi+\pi).R_X\left(\frac{\pi}{2}\right).`
             :math:`R_Z(\theta+\pi).S_X\left(\frac{\pi}{2}\right).U_1(\lambda)`
+        * - 'ZSXX'
+          - :math:`Z(\phi) Y(\theta) Z(\lambda)`
+          - :math:`e^{i\gamma} U_1(\phi+\pi).R_X\left(\frac{\pi}{2}\right).`
+            :math:`R_Z(\theta+\pi).S_X\left(\frac{\pi}{2}\right).U_1(\lambda)`
```

Comment:
Here you probably should use RZ instead of U1.
Also where does X appear?

Reply:
> Ok after the comments I feel better that this will likely not reintroduce platform-dependent issues. But I'm curious why this was tagged as 0.17 milestone (which is a few days away)? I think doing this is less risky after release.
> 
> Other than that I like the direction of avoiding unnecessary roundings here. But I do have a concern about the code refactor in the 1Q decomposition code that I commented.

I don't think this needs to be release blocking on 0.17, but I also think its safe enough to merge for 0.17.

Reply:
This carried over from ZSX so I'll fix both